pages/dashboard/index.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import _ from 'lodash';
+import getConfig from 'next/config';
+import styled from 'styled-components';
+import { useTranslation } from 'react-i18next';
+
+import { useAuth } from '../../lib/Auth';
+import { useMatrix } from '../../lib/Matrix';
+import ServiceInvitations from './ServiceInvitations';
+import { ServiceTable } from '../../components/UI/ServiceTable';
+import DisplayInvitations from './DisplayInvitations';
+
+const sortRooms = function(room) {
+    return [
+        room.notificationCount === 0,
```

Comment:
Doesn't really make sense to sort them by the notification count --- pending invitations always have a `notificationCount` of `1`, I guess. At least it's always the same.

I think it would make more sense to replace the `_.sortBy([...matrix.invites.values()], sortRooms)` line with something like: `_.sortBy([...matrix.invites.values()], 'name')` instead.